.gitignore

```diff
@@ -9,6 +9,7 @@ kbfstool/kbfstool
 kbfsfuse/revision
 kbfsgit/git-remote-keybase/git-remote-keybase
 kbpagesd/kbpagesd
+kbpagesconfig/kbpagesconfig
```

Comment:
We should build both `kbpagesd` and `kbpagesconfig` in `appveyor.yml`, since I think it only builds/installs specific directories.

Reply:
Good point; I'll add the tests to Jenkins as well.